Automattic/wp-calypso
# Add/descriptions to domains

#### Changes proposed in this Pull Request

This PR adds the domain type to the site-level meta field on the domain list. 

**Before**
![image](https://user-images.githubusercontent.com/6981253/91515905-3e980c80-e8b8-11ea-9b5b-9a9342e2154f.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/91515843-13adb880-e8b8-11ea-9a3b-bb5b52a41def.png)

#### Testing instructions
- Visit the domain list at the site level to confirm the right labels.
- Visit all domains list to make sure everything still looks ok.
